### DISCUSSION TITLE
Measurement Tab in iTwin Viewer 3.x

### DISCUSSION THREAD

**Discussion by ashish-srivastava1** [iTwin/itwinjs-core#3845] [General]
Is there a way to hide Measurement Tab in iTwin Viewer?

![Measurement-tab](https://user-images.githubusercontent.com/73111819/175007721-06c657ce-f534-4694-8272-97e18d14c6f1.png)

**Comment by aruniverse** [comment]
If you upgrade to the latest itwin/measure-tools-react PKG you should be able to auto hide the widget unless a measurement has been created/selected.

**Comment by achrysaetos** [comment]
Also make sure your @itwin/web-viewer-react version is at least 2.0.10 or above. Otherwise, there will be a conflict between two versions of measure-tools-react in your package-json.lock file, because if your @itwin/web-viewer-react version is 2.0.5 for example, it would depend on measure-tools-react version 0.2.0, which is outdated.
